### PR TITLE
Tmp: Feedback

### PR DESCRIPTION
**Pull request by sanderpick**
Opening this so I can add some comments to unchanged lines...

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,16 +4,15 @@ go 1.13
...
 	github.com/golang/protobuf v1.3.1
-	github.com/hsanjuan/ipfs-lite v0.1.6
+	github.com/hsanjuan/ipfs-lite v0.1.7
 	github.com/ipfs/go-block-format v0.0.2
 	github.com/ipfs/go-cid v0.0.3
 	github.com/ipfs/go-ipld-cbor v0.0.3
 	github.com/ipfs/go-ipld-format v0.0.2
 	github.com/ipfs/go-log v0.0.1
 	github.com/ipfs/go-merkledag v0.2.3
-	github.com/libp2p/go-libp2p-crypto v0.1.0
```
Comment (sanderpick) [inline go.mod:14]: deprecated

File: mobile/mobile.go [hunk 2]
```diff
@@ -15,7 +15,9 @@ func Start(datastorePath string) error {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	log.SetLogLevel("*", "warn")
+	if err := log.SetLogLevel("*", "warn"); err != nil {
```
Comment (sanderpick) [inline mobile/mobile.go:18]: handle error

File: server/server.go [hunk 3]
```diff
@@ -92,10 +92,10 @@ func (s *ipfsLiteServer) AddFile(srv pb.IpfsLite_AddFileServer) error {
 	for {
 		req, err := srv.Recv()
 		if err == io.EOF {
-			writer.Close()
+			_ = writer.Close()
```
Comment (sanderpick) [inline server/server.go:95]: generally a good idea to explicitly ignore unhandled errors (your IDE linter should be underlining these)

File: server/server.go [hunk 4]
```diff
@@ -123,12 +123,12 @@ func (s *ipfsLiteServer) AddFile(srv pb.IpfsLite_AddFileServer) error {
 }
 
 func (s *ipfsLiteServer) GetFile(req *pb.GetFileRequest, srv pb.IpfsLite_GetFileServer) error {
-	cid, err := cid.Decode(req.GetCid())
```
Comment (sanderpick) [inline server/server.go:126]: `cid` conflicts with `cid` package (linter should underline these)

File: server/server_test.go [hunk 5]
```diff
@@ -9,32 +9,40 @@ import (
...
 	"google.golang.org/grpc"
 )
 
 var (
+	pctx context.Context
+	pcancel context.CancelFunc
+
+	litepeer                                                   *ipfslite.Peer
 	client                                                     pb.IpfsLiteClient
-	stringToAdd                                                string = "hola"
```
Comment (sanderpick) [inline server/server_test.go:24]: unneeded type

File: server/server_test.go [hunk 6]
```diff
@@ -9,32 +9,40 @@ import (
...
+	pctx, pcancel = context.WithCancel(context.Background())
+
+	var err error
+	litepeer, err = newPeer(pctx)
 	if err != nil {
 		t.Fatalf("failed to create peer: %v", err)
 	}
 
-	go StartServer(peer, "localhost:10000")
+	go StartServer(litepeer, "localhost:10000")
```
Comment (sanderpick) [inline server/server_test.go:45]: allow the use of the peer package (no name conflict)

File: server/server_test.go [hunk 7]
```diff
@@ -291,7 +313,7 @@ func TestGetNodes(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to GetNodes: %v", err)
 	}
-	results := []*pb.Node{}
+	var results []*pb.Node
```
Comment (sanderpick) [inline server/server_test.go:316]: cleaner way to init a slice (the default value of a slice is empty)

File: server/server_test.go [hunk 8]
```diff
@@ -454,16 +476,22 @@ func TestRemoveNodes(t *testing.T) {
 	}
 }
 
-func newPeer() (*ipfslite.Peer, error) {
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
+func TestClose(t *testing.T) {
+	pcancel()
```
Comment (sanderpick) [inline server/server_test.go:480]: close context when all tests are done

File: server/server_test.go [hunk 9]
```diff
@@ -454,16 +476,22 @@ func TestRemoveNodes(t *testing.T) {
...
+}
 
-	log.SetLogLevel("*", "warn")
+func newPeer(ctx context.Context) (*ipfslite.Peer, error) {
+	_ = log.SetLogLevel("*", "debug")
+
+	// Bootstrappers are using 1024 keys. See:
+	// https://github.com/ipfs/infra/issues/378
+	crypto.MinRsaKeyBits = 1024
```
Comment (sanderpick) [inline server/server_test.go:488]: (this was using the deprecated crypto package)

File: go.mod (hunk 1 above)
Comment (sanderpick) [reply go.mod:14]: it's handy to run `go mod tidy` after `go get <new or updated package>`, which cleans out unneeded stuff from mod and sum files.

Comment (asutula) [reply go.mod:14]: do you mean this entire lib is deprecated, or this version is outdated?

Comment (sanderpick) [reply go.mod:14]: lib is deprecated in favor of go-libp2p-core/crypto

Comment (asutula) [reply go.mod:14]: ah I see, thx

**Pull request closed without merging**